Misc/NEWS.d/next/Library/2019-06-22-12-30-00.bpo-37372.kIKqZ6.rst

```diff
@@ -0,0 +1 @@
+Fix errors unpickling certain datetime.time objects from Python 2.
```

Comment:
Please add a little more details: "with minutes or seconds >= 24".

Add also "Patch by Justin Blanchard." here and add your name in Misc/ACKS.